<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameEditWindow</class>
 <widget class="QDialog" name="GameEditWindow">
  <property name="windowTitle">
   <string>Добавление/Редактирование Игры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Добавление/Редактирование Игры</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="descriptionTextEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="publisherLabel">
       <property name="text">
        <string>Издатель:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="publisherLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="releaseYearLabel">
       <property name="text">
        <string>Год выпуска:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="releaseYearSpinBox">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2023</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="playersLabel">
       <property name="text">
        <string>Игроки:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="playersSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="genresLabel">
       <property name="text">
        <string>Жанры:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QListWidget" name="genresListWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
